modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro

```diff
@@ -132,7 +132,7 @@ definition {
 		}
 
 		if (!(isSet(shortDescription))) {
-			var shortDescription = "";
+			var shortDescription = "${shortDescription}";
```

Comment:
@baharturk can you please revert this one? This should remain as it is now on master

Reply:
@baharturk you can use macro CommerceProducts.searchAndGoToProductEntry